Atmosphere guide (#57)
eradiate/eradiate: docs/rst/user_guide/atmosphere/absorption.rst

.. _sec-atmosphere-molecular-absorption:

Molecular absorption
====================

There are several ways to compute molecular absorption.
The most general and accurate way is the *monochromatic model*, also known as
the *line-by-line model*.
There are also approximate models, such as the *CK model*, which are less
accurate but faster.
:cite:`Taine2014TransfertsThermiquesIntroduction`.

Monochromatic model
-------------------

In monochromatic mode, the monochromatic model is used.
In the monochromatic model, the air monochromatic absorption coefficient,
:math:`k_{\mathrm a \lambda} \, [L^{-1}]`,
due to :math:`N` absorbers, is computed using the equation:

.. math::
   :label: mono_k

   k_{\mathrm a \lambda} (p, T, \vec{x}) = \sum_{i=0}^{N-1} \, x_i \, n \,

>    \sigma_{\mathrm{ai}}(\nu, p, T)

Reviewer: ```suggestion
   \sigma_{\mathrm{a}i}(\nu, p, T)
```
I assume :) (If I'm right, to be fixed in the rest of the document)

where

* :math:`\lambda` is the wavelength :math:`[L]` (subscript indicates spectral
  dependence),
* :math:`p` is the pressure :math:`[ML^{-1}T^{-2}]`,
* :math:`T` is the temperature :math:`[\Theta]`,
* :math:`\vec{x}` is the absorbers volume fraction vector, whose components are
  the individual absorbers volume fractions in air, :math:`x_i` :math:`[/]`,
* :math:`n` is the air number density :math:`[L^{-3}]`,
* :math:`\sigma_{\mathrm {ai}}` is the monochromatic absorption cross section
  :math:`[L^2]`, of the :math:`i` -th absorber and
* :math:`\nu = \lambda^{-1}` is the wavenumber :math:`[L^{-1}]`,

.. note::

>    The sum of the absorbers volume fractions is not necessarily 1, because there
>    are some non-absorbing species in the air.

Reviewer: ```suggestion
   The sum of the absorbers volume fractions can be less than 1 due to the presence 
   of non-absorbing species in the air.
```
